platformio.ini

```diff
@@ -13,7 +13,7 @@ board_f_cpu = 160000000L
 lib_deps_builtin =
   SPI
 lib_deps_external =
-  sidoh/RF24
+  sidoh/RF24#remove_progmem
```

Comment:
Should probably remove this.  Will definitely want to figure out what's going on with Windows builds for the v1.6.0 release, but hoping this won't be the fix.

Reply:
Yes, maybe a good ideal to NOT have this in your branch. But I needed this. :)
Maybe I find some time to set up the build environment on my MacBook...